packages/tradershub/src/features/cfd/modals/MT5PasswordModal/SubmitHandler.jsx

```diff
@@ -0,0 +1,58 @@
+import { Category, MarketType, PlatformDetails } from '../../constants';
+import { Jurisdiction } from '../../screens/CFDCompareAccounts/constants';
+
+export const submitHandler = async (
+    isMT5PasswordNotSet,
+    tradingPasswordChange,
+    password,
+    marketType,
+    activeTrading,
+    mutate,
+    settings,
+    selectedJurisdiction,
+    availableMT5Accounts
+) => {
+    const accountType = marketType === MarketType.SYNTHETIC ? 'gaming' : marketType;
```

Comment:
Can use constant file here for `gaming` but its upto you 
Its a small nit but it is hardcoded value